uint256.go

```diff
@@ -673,6 +673,29 @@ func (z *Int) MulMod(x, y, m *Int) *Int {
 	return z.Set(&rem)
 }
 
+// MulDivOverflow calculates (x*y)/d with full precision, returns z and whether overflow occurred.
```

Comment:
Please explain what "overflow" means in this case: the division result does not fit 256-bit return type.